src/components/pages/PageDecksCreate/PageDecksCreate.jsx

```diff
@@ -0,0 +1,252 @@
+
+  const populateConflicts = useCallback(
+    async () => {
+      try {
+        const cardRefs = await firebase.db
+          .collection('cards')
+          .where('createdBy', '==', user.uid)
+          .where('type', '==', 'Conflict')
+          .get();
+
+        const newConflictOptions = cardRefs
+          .docs
+          .map((card) => {
+            const cardData = {
+              id: card.id,
+              ...card.data(),
+            };
+            return { label: cardData.cardTitle, value: cardData.id };
+          });
+
+        setConflictOptions(newConflictOptions);
+        setIsLoading(true);
+      } catch (err) {
+        console.error(err);
+      }
+    },
+    [user.uid],
+  );
+
+  function changeDeck(key, value) {
+    const newDeck = { ...deck };
+    newDeck[key] = value;
+    setDeck(newDeck);
+  }
+
+  async function handleCreateCard() {
+    const newDeck = {
+      name: deck.name,
+      description: deck.description,
+      characterCards: selectedCharacterIds.map((cardId) => (
+        {
+          cardRef: firebase.db.collection('cards').doc(cardId.value),
+          quantity: 1,
+        }
+      )),
+      circumstanceCards: selectedCircumstanceIds.map((cardId) => (
+        {
+          cardRef: firebase.db.collection('cards').doc(cardId.value),
+          quantity: 1,
+        }
+      )),
+      conflictCards: selectedConflictIds.map((cardId) => (
+        {
+          cardRef: firebase.db.collection('cards').doc(cardId.value),
+          quantity: 1,
+        }
+      )),
+      visibility: 'public',
+      createdBy: user.uid,
+    };
+
+    try {
+      await firebase.db.collection('decks').add(newDeck);
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error(err);
+    }
+  }
+
+  useEffect(() => {
+    populateCharacters();
+    populateCircumstances();
+    populateConflicts();
+  }, [populateCharacters, populateCircumstances, populateConflicts]);
+
+  return (
+    <TemplateDefault data-test="p-create-decks">
+      <h1>Create A Deck</h1>
+      <If condition={!isLoading}>
+        <LoadingAnim />
+
+        <Else>
+          <form
+            data-test="p-decks-create__submit"
+            onSubmit={(e) => {
+              e.preventDefault();
+              handleCreateCard();
+              history.push('/decks');
+            }}
+          >
+            <div className="form-group">
+              <label htmlFor="name">
+                Name: &nbsp;
+              </label>
+              <input
+                data-test="p-decks-create__name"
+                value={deck.name}
+                onChange={(e) => {
+                  changeDeck('name', e.target.value);
+                }}
+                className="form-control"
+                id="name"
+                required="required"
+              />
+            </div>
+
+            <div className="form-group">
+              <label htmlFor="description">
+                Description: &nbsp;
```

Comment:
Don't think you need an &nbsp;. Use `mr-2` or something instead if you really want that.